web/src/test/java/com/replica/RouterServerTest.java

```diff
@@ -68,15 +68,16 @@ public class RouterServerTest extends ReplicaGraphHopperTest {
     private static final double[] REQUEST_ORIGIN_2 = {38.59337420024281, -121.48746937746185}; // Sacramento area
     private static final double[] REQUEST_DESTINATION_1 = {38.55518457319914, -121.43714698730038}; // Sacramento area
     private static final double[] REQUEST_DESTINATION_2 = {38.69871256445126, -121.27320348867218}; // South of Roseville
+    private static final double[] REQUEST_DESTINATION_3 = {38.68163283946138,-121.15723016671839}; // Folsom
```

Comment:
Just curious as to what is special about this destination.

Reply:
Nothing in particular, I just needed a park and ride request that resulted in a transfer (for the relevant test that covers this case), and I couldn't recreate one with any of the other existing origins + destinations after the beta knobs fix changed routing behavior a bit